lib/listener/helpers.js

```diff
@@ -13,6 +13,10 @@ module.exports = function () {
   const runHelpersHook = (hook, param) => {
     if (store.dryRun) return;
     Object.keys(helpers).forEach((key) => {
+      if (!helpers[key][hook]) {
+	    console.log(`${key}'s hook value is: ${helpers[key][hook]}`);
```

Comment:
please use `output.log` from `require('../output')`;
you will be able to see that message when executing in `--verbose` mode.

Also you can use `debug` calls from `debug` module
But not console.log!